src/departure/index.js

```diff
@@ -0,0 +1,171 @@
+// @flow
+import { journeyPlannerQuery } from '../api'
+import {
+    BUS, TRAM, RAIL, METRO, WATER, AIR, COACH, CAR,
+} from '../constants/travelModes'
+
+import {
+    getDeparturesFromStopPlacesQuery,
+    getDeparturesFromQuayQuery,
+    getDeparturesBetweenStopPlacesQuery,
+} from './query'
+
+import type {
+    StopPlaceDepartures,
+    QuayDepartures,
+    Departure,
+} from '../../flow-types/Departures'
+
+
+type GetDeparturesParams = {
+    includeNonBoarding?: boolean,
+    limit?: number,
+    departures?: number, // deprecated
+    start?: Date,
+    timeRange?: number,
+}
+export function getDeparturesFromStopPlaces(
+    stopPlaceIds: Array<string>,
+    params?: GetDeparturesParams = {},
+): Promise<Array<StopPlaceDepartures>> {
+    const {
+        limit = 50,
+        departures,
+        timeRange = 72000,
+        start = new Date(),
+        includeNonBoarding = false,
+        ...rest
+    } = params
+
+    if (departures !== undefined) {
+        // eslint-disable-next-line no-console
+        console.info('Entur SDK: "departures" is deprecated, use "limit" instead.')
```

Comment:
Det er kanskje ikkje nødvendig med deprecation-warning for departures, når navnet på funksjonen (getDeparturesFromStopPlaces) er endra. Då kan me jo tenke på dette som ein ny funksjon, med eit nytt API.

Reply:
smart, fjerna det jeg